components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementService.java

```diff
@@ -35,5 +35,10 @@
 
     public void unregisterEventSync(String syncId);
 
+    public void updateLatestEventSentTime(String publisherName, int tenantId, long timestamp);
+
+    public long getLatestEventSentTime(String publisherName, int tenantId);
+
+    public long getClusterTimeInMilies();
```

Comment:
Thanks @nirmal070125, changes are done in pull request - https://github.com/wso2/carbon-analytics-common/pull/121